File: app/modules/IOrderModule.java
package modules;

import models.IEndUser;
import models.IOrder;
import models.StandardReturn;

import java.util.List;

/**
 * Created by Beemen on 04/08/2015.
 */
public interface IOrderModule {
    StandardReturn SubmitOrder(IOrder order, IEndUser session);
    List<IOrder> GetUserOrders(IEndUser user);
}

File: Modules/IOrderModule.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using end_user_gui.Models;

namespace end_user_gui.Modules
{
    public interface IOrderModule
    {
        StandardReturn SubmitOrder(IOrder order, EndUser session);
        List<IOrder> GetUserOrders(EndUser user);
    }
}